generators/client/templates/src/main/webapp/app/components/login/_login.controller.js

```diff
@@ -48,9 +48,8 @@
 
                 // If we're redirected to login, our
                 // previousState is already set in the authExpiredInterceptor. When login succesful go to stored state
-                if ($rootScope.redirected && $rootScope.previousStateName) {
-                    $state.go($rootScope.previousStateName, $rootScope.previousStateParams);
-                    $rootScope.redirected = false;
```

Comment:
I forgot why I added this but it was done for some reason. Are you sure everything works fine without this?